File: Safari-609.1.11.1/LayoutTests/http/tests/security/contentSecurityPolicy/report-only-from-header.php
<?php
header("Content-Security-Policy-Report-Only: script-src 'self'; report-uri resources/save-report.php");
?>
<script>
// This script block will trigger a violation report but shouldn't be blocked.
alert('PASS');
</script>
<script src="resources/go-to-echo-report.js"></script>

File: LayoutTests/http/tests/security/contentSecurityPolicy/report-only-from-header.py
#!/usr/bin/env python3

import sys

sys.stdout.write(
    'Content-Security-Policy-Report-Only: script-src \'self\'; report-uri resources/save-report.py\r\n'
    'Content-Type: text/html\r\n\r\n'
    '<script>\n'
    '// This script block will trigger a violation report but shouldn\'t be blocked.\n'
    'alert(\'PASS\');\n'
    '</script>\n'
    '<script src="resources/go-to-echo-report.js"></script>\n'
)